modules/standard/List.chpl

```diff
@@ -188,6 +212,24 @@ module List {
       this._firstTimeInitializeArrays();
     }
 
+    /*
+      Initializes a list containing elements that are copy initialized from
+      the elements contained in another list.
+
+      Used in new expressions.
+
+      :arg other: The list to initialize from.
+    */
+    proc init(other: list(?t)) {
```

Comment:
Can this initializer's body be `this.init(other, false)` ?

The default for parSafe contradicts the module comment "When constructed from another list, the new list will inherit the parallel safety mode of its originating list." Please reconcile.

Why map and set do not have an analogous initializer?